src/Microsoft.AspNetCore.JsonPatch/JsonPatchDocumentOfT.cs

```diff
@@ -93,7 +93,7 @@ public JsonPatchDocument<TModel> Add<TProp>(Expression<Func<TModel, TProp>> path
 
             Operations.Add(new Operation<TModel>(
                 "add",
-                ExpressionHelpers.GetPath(path).ToLowerInvariant() + "/" + position,
+                ExpressionHelpers.GetPath(path, ContractResolver).ToLowerInvariant() + "/" + position,
```

Comment:
I'm suggesting that we get rid of the 'helpers' because it's really just the core functionality of this class by another name.